format-specs/geoparquet.md

```diff
@@ -50,57 +50,116 @@ Version of the geoparquet spec used, currently 0.3.0
+            "abbreviation": "Lon",
+            "direction": "east",
+            "unit": "degree"
+        },
+        {
+            "name": "Geodetic latitude",
+            "abbreviation": "Lat",
+            "direction": "north",
+            "unit": "degree"
+        }
+        ]
+    },
+    "id": {
+        "authority": "OGC",
+        "code": "CRS84"
+    }
+}
+```
 
-The CRS must be provided in [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_coordinate_reference_systems) version 2, also known as **WKT2**. WKT2 has several revisions, this specification only supports [WKT2_2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html).
+OGC:CRS84 is equivalent to the well-known
+[EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from
+latitude-longitude to longitude-latitude.  EPSG:4326 and OGC:CRS84 are
+equivalent with respect to this specification because this specification
+specifically overrides the coordinate axis order of the stored coordinates to be
+longitude-latitude.
 
-If CRS is not provided, then all coordinates in the geometry must use longitude, latitude to store their data.
-If an implementation is CRS-aware and needs a CRS representation of the data it should assume a default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). It's equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude. The WKT2:2019 string for OGC:CRS84 is:
+Implementations that are not CRS-aware and operate entirely with longitude,
+latitude coordinates may be able to infer that coordinates conform to the
+OGC:CRS84 CRS based on elements of the `crs` field.  For simplicity, Javascript
+object dot notation is used to refer to nested elements.
 
-```
-GEOGCRS["WGS 84 (CRS84)",
-    ENSEMBLE["World Geodetic System 1984 ensemble",
-        MEMBER["World Geodetic System 1984 (Transit)"],
-        MEMBER["World Geodetic System 1984 (G730)"],
-        MEMBER["World Geodetic System 1984 (G873)"],
-        MEMBER["World Geodetic System 1984 (G1150)"],
-        MEMBER["World Geodetic System 1984 (G1674)"],
-        MEMBER["World Geodetic System 1984 (G1762)"],
-        MEMBER["World Geodetic System 1984 (G2139)"],
-        ELLIPSOID["WGS 84",6378137,298.257223563,
-            LENGTHUNIT["metre",1]],
-        ENSEMBLEACCURACY[2.0]],
-    PRIMEM["Greenwich",0,
-        ANGLEUNIT["degree",0.0174532925199433]],
-    CS[ellipsoidal,2],
-        AXIS["geodetic longitude (Lon)",east,
-            ORDER[1],
-            ANGLEUNIT["degree",0.0174532925199433]],
-        AXIS["geodetic latitude (Lat)",north,
-            ORDER[2],
-            ANGLEUNIT["degree",0.0174532925199433]],
-    USAGE[
-        SCOPE["Not known."],
-        AREA["World."],
-        BBOX[-90,-180,90,180]],
-    ID["OGC","CRS84"]]
-```
+The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id`
```

Comment:
Thanks for putting this together. I have some thoughts on it, but seemed to be a bigger scope than just a line in a PR, getting into if we might be able to tweak projjson. See https://github.com/opengeospatial/geoparquet/discussions/98